exchanges/retry/src/retryExchange.ts

```diff
@@ -75,14 +75,23 @@ export const retryExchange = ({
           })
         );
 
+        client.debugTarget!.dispatchEvent({
+          type: 'retryRetrying',
+          message: 'Retrying the operation',
```

Comment:
I think the verbosity of the other message below would be more useful.

I'm not totally sure what the cause of the retry is but would something like this work?

```suggestion
          message: `A network request has failed and a retry has been triggered (${retryCount} / ${MAX_ATTEMPTS})`,
```